rsqueakvm/storage_contexts.py

```diff
@@ -311,8 +321,7 @@ def store_stackpointer(self, size):
             assert size >= 0, "trying to store negative stackpointer"
             self.pop_n(depth - size)
         else:
-            for i in range(depth, size):
-                self.push(self.space.w_nil)
+            self.store_stack_ptr(size)
```

Comment:
There's already something on the stack (probably nil), it's fine to leak stuff also, there is not guarantee that the stack is nil-ed